Neos.Flow/Classes/Testing/RequestHandler/EmptyRequestHandler.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/*
+ * (c) Contributors of the Neos Project - www.neos.io
+ * Please see the LICENSE file which was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Neos\Flow\Testing\RequestHandler;
+
+use Neos\Flow\Annotations as Flow;
+use Neos\Flow\Core\RequestHandlerInterface;
+
+/**
+ * A test request handler that does absolutely nothing
+ *
+ * @Flow\Proxy(false)
+ */
+class EmptyRequestHandler implements RequestHandlerInterface
```

Comment:
can't this be `final`
```suggestion
final class EmptyRequestHandler implements RequestHandlerInterface
```